content/departments/engineering/teams/integrations/integrations-support-rotation.md

```diff
@@ -0,0 +1,58 @@
+
+You are not expected to provide all answers to inquiries or review all PRs. You are expected to be the initial responder and decide whether to tackle the work right away or add it for later planning. As a general rule of thumb, any work that will take longer than half a day should become an issue in the team's [GitHub board](https://github.com/orgs/sourcegraph/projects/213/views/18). This also includes work that takes less than half a day but needs to be transitioned to another engineer.
+
+The engineer on support rotation is responsible for anything that doesn't have a clear owner in our team. For example if we start receiving a trove of PRs for something that another engineer is currently working on, there's no need to keep an eye on those. On the other hand, if it's a one-off review then it's your responsibility to ensure the ball doesn't get dropped.
+
+Finally, it's your responsibility to ensure a smooth transition to the next engineer on rotation. This includes:
+
+- Wrapping up tasks as you can
+- Ensuring all tasks are properly documented (with GitHub issues and status updates)
+- Communicating any leftover tasks/asks/things to keep in mind for whoever is up next. This is best done the day before the rotation transitions via an async recap update posted in `#integrations-internal`.
+
+### Channels to keep an eye on
+
+The following channels/boards should be checked at least _once a day_:
+
+- #integrations
+- #integrations-internal
+- #integrations-ide
+- Messages that tag `@integrations-team` or `@integrations-eng`
+- GitHub notifications tagging `@sourcegraph/integrations`
+- GitHub issues that tag `team/integrations`.
+
+If you're curious, here's a work-in-progress list of [all channels that support tickets can come from](https://docs.google.com/document/d/1cxNVXZIn1FEqrPCEOQrYHQAn2EiLjyY4oxWE99SyK9Q/edit).
+
+## Why we are doing this
+
+This process was first discussed as a suggestion to improve how we handle feedback in our team during the 2022 Barcelona team offsite.
+
+We believe this process has the following benifits:
+
+- Engineers have easier exposure to all areas that the team owns (everyone will eventually see BExt, VSCE, JB and other bugs). This will also help with onboarding new engineers.
+- We can rely on incoming feedback being responded to, so others have piece of mind and can focus on their work.
+- We have an explicit time budget so the person on support rotation knows that they will have the time to work on triaging.
+- We can set up SLAs on various channels.
+
+## Current schedule
+
+| Time frame    | Engineer |
+| ------------- | -------- |
+| Sept 26–Oct 3 | Philipp  |
+| Oct 3–Oct 10  | David    |
+| Oct 10–Oct 17 | Taras    |
+| Oct 17–Oct 24 | Beatrix  |
+| Oct 24–Oct 31 | Philipp  |
+| Oct 31–Nov 7  | David    |
+| Nov 7–Nov 14  | Taras    |
+| Nov 14–Nov 21 | Beatrix  |
```

Comment:
Yeah I agree that this is not great. Ideally it would be in the calendar I think so that you can plan around it (i.e if you book vacation during your support schedule, it should be easily possible to swap with someone else). A bit of plan-ability would be nice though so I don't want to rely on it just for Slack.

Reply:
I propose having a team calendar where it'd be easier for all of us to add events and keep track of team-related deadlines instead of having to check individual calendar ✋

Reply:
I'll look into this right away 👍